api/events.proto

```diff
@@ -43,6 +43,7 @@ message Event {
     string remote_ip = 4 [(gogoproto.customname) = "RemoteIP"];
   }
   Authentication authentication = 9;
+  string user_agent = 10;
```

Comment:
I have to say it feels a bit weird to now have `remote_ip` inside `authentication` and `user_agent` separate. Maybe we should pull `remote_ip` out as well then.

I think doing this wouldn't violate our compatibility commitment, because (1) events aren't covered and (2) because of the bug you're fixing in #3068 we weren't sending this yet anyway.

Reply:
What about the opposite? Adding the `user_agent` inside `authentication`? This maintains compatbility (even if not explicitly required by the compatibility commitment)

Reply:
Because users are not authenticating with an IP address or user agent.